README.md

```diff
@@ -9,12 +9,57 @@ $ yarn install
 # serve with hot reload at localhost:3000
 $ yarn dev
 
+# generate static project
+$ yarn generate
+
 # build for production and launch server
 $ yarn build
 $ yarn start
+```
 
-# generate static project
-$ yarn generate
+## Run Tests 🧪
+```bash
+# run all tests with jest
+$ yarn jest
+# If making UI changes make sure to update related snapshots.
+$ yarn jest -u
 ```
 
-For detailed explanation on how things work, check out [Nuxt.js docs](https://nuxtjs.org).
+## Test Frontend Setup 🐥
```

Comment:
We should also explain that when running on kovan you need to add your mnemonic to `mnemonic.txt` and make sure that account number 0 in your wallet has enough funds for the respective action / task.

Reply:
Do you suppose this will be something the judges would look at? I definitely think is a great suggestion overall though and would make the project stronger.

Reply:
I don't know much at all about how hackathons are judged. But I would like to enable the judges as much as possible to reproduce whatever we will be showing in the demo, so for that purpose we would need to document these tasks.

Reply:
Now that we have moved the remote node url to a gitignored url.txt file we should also tell the user that he has to create that file and save his full infura / alchemy url in that file.